src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -900,7 +900,7 @@ export const WalletProvider = ({ children }: { children: React.ReactNode }): JSX
     })
   }, [])
 
-  useEffect(() => load(), [load, state.adapters, state.keyring])
+  useEffect(() => load(), [load, state.keyring])
```

Comment:
We don't want to run this reactively on `state.adapters` anymore after this stack/PR refactor:

1. This stack's refactor makes it so that we `getAdapter()` so we don't need to be reactive on `state.adapters` anymore
2. This PR's refactor moved some initialization logic in each switch case instead of the previous outer scope, meaning we would have infinite dispatch and reactivity after the dispatch of `WalletActions.SET_ADAPTERS` in `load()`